test/integration/backward_compatible/sql/ExecuteTest.js

```diff
@@ -538,7 +545,14 @@ describe('SqlExecuteTest', function () {
             });
             error1.should.be.instanceof(getHazelcastSqlException());
             error1.code.should.be.eq(getSqlErrorCode().CONNECTION_PROBLEM);
-            error1.originatingMemberId.should.be.eq(client.connectionManager.getClientUuid());
+            // There was a fix regarding originatingMemberId in 5.0 and 4.2.x after 4.2.0.
+            // TODO: Update here to be 4.2.1 if we release 4.2.1
+            // https://github.com/hazelcast/hazelcast-nodejs-client/pull/940
+            if (TestUtil.isClientVersionAtLeast('5.0.0')) {
+                error1.originatingMemberId.should.be.eq(client.connectionManager.getClientUuid());
+            } else {
+                error1.originatingMemberId.should.be.eq(member.uuid);
```

Comment:
should we need toString here? @mdumandag see https://github.com/srknzl/test/runs/2904025971?check_suite_focus=true

Reply:
This is the updated version that passes:

https://github.com/srknzl/test/runs/2904280680?check_suite_focus=true

see tests branch in srknzl/test for my latest commit